### ISSUE TITLE
Publish Endpoint locations in the documentation?

### ISSUE THREAD

**Issue by StevenACoffman** [argodev/codemash-speakers#1]
So I'm not sure if this repo is actually affiliated with the CodeMash Conference organizers or not. I was able to use your html examples to extract the endpoint locations for my own purposes (thanks!), but I'm not sure if distributing them more widely (in the documentation) would wind up running up your Azure bills. 

Also, as there may be last minute changes to the session schedule (cancellations, room changes) I'm not sure if the data in these endpoints is going to be changed to be accurate or not.

**Comment by argodev** [comment]
Yes, I am one of the CodeMash organizers and this is the "official" documentation... I'll work on updating the docs to actually list the endpoints... that was an oversight on my part.

The endpoints listed are the "ground truth" for any changes occurring at the conference, so you are safe to use them and know that any changes will be reflected there.

Thanks

**Comment by StevenACoffman** [comment]
Neat! If it is helpful, I used your api and data, converted it to ICS so I could put all the sessions into my phone more conveniently.

You can get just the [ICS of all sessions here](https://rawcdn.githack.com/StevenACoffman/CodeMash2iCalendar/master/events/CODEMASH2018.ics) or the code to re-generate it (and also individual ics files per each session) in [my repo over here](https://github.com/StevenACoffman/CodeMash2iCalendar). My documentation is currently horrible, so you're doing better than me.
